CONTRIBUTING.md

```diff
@@ -11,3 +11,7 @@ with how you'd like to contribute, if you want to introduce yourself.
 
 Feature planning is done on a Trello board; please open an issue and mention
 [@cheshire137](https://github.com/cheshire137) to get added.
+
+Please know we have a [Code of
+Conduct](https://github.com/cheshire137/overwatch-team-comps/blob/master/CODE_OF_CONDUCT.md) in place.
```

Comment:
I think you can just put `CODE_OF_CONDUCT.md` inside the parentheses and it will work as a relative link.